CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## 0.5.0 - January 26, 2024
+
+- **Breaking Changes** A new `onWebPixelEvent(PixelEvent)` function has been added to the `CheckoutEventProcessor` interface. This allows listening for [Web Pixel](https://shopify.dev/docs/apps/marketing/pixels) events that take place in checkout, so they can be emitted to your preferred analytics system. See `README.md` for more details.
+
+_Note_: If your processor extends `DefaultCheckoutEventProcessor`, a no-op implementation has been added, so no changes are required unless you'd like to respond to pixel events. If your processor does not extend `DefaultCheckoutEventProcessor`, you will need to implement this function.
```

Comment:
The problem is that it's not mandatory to extend `DefaultCheckoutEventProcessor`.  Hopefully most folks are, but otherwise it'll be breaking. 

What do you think, maybe the _Note_ is enough and we can drop `**Breaking Changes**

Reply:
Ah gotcha, it's fine as is then